src/app/shared/modal/modal-login/modal-login.component.ts

```diff
@@ -36,19 +36,28 @@ export class ModalLoginComponent implements OnInit {
 
   login(): void {
     this.loadingService.loadingSubject.next(true);
```

Comment:
Such operation are better to be done in the service
Pass the true
Create the request
In the pipe use the RxJs operator finalize - and pass false

And have you created UI for spinner to show the loading status?

Reply:
I have refactored it with the getter/setter and private field of loadingSubject. So for now I change the subject with next() method only in the service when that subject was created. 
Sure, I created spinner. Here is the LoadingBlockComponent